conceptnet5/vectors/evaluation/bias.py

```diff
@@ -0,0 +1,197 @@
+import numpy as np
+import pandas as pd
+import scipy
+
+from conceptnet5.vectors import standardized_uri, normalize_vec
+from conceptnet5.vectors.transforms import (
+    l2_normalize_rows, subtract_mean_vector
+)
+from conceptnet5.vectors.debias import (
+    FEMALE_WORDS, MALE_WORDS, PEOPLE_BY_BELIEF, PEOPLE_BY_ETHNICITY,
+    CULTURE_PREJUDICES, get_category_axis, get_vocabulary_vectors
```

Comment:
Aha, right.

I currently have something commented out that calculates `trained_ethnic_bias`, where it evaluates on `CULTURE_PREJUDICES`, which is exactly the correlation that it was trained to remove.

I found this useful as a test that bias removal was working, but not as a number that made any sense to compare between systems, so I commented it out. I guess I should come to a more solid conclusion about what to do with that measurement than just having it commented out.